common/msg.c

```diff
@@ -322,12 +322,24 @@ static void set_term_color(void *talloc_ctx, bstr *text, int c)
         return;
     }
 
-    bstr_xappend_asprintf(talloc_ctx, text, "\033[%d;3%dm", c >> 3, c & 7);
+    // Promote black/white to bright ones
+    if (c == 0 || c == 7)
+        c += 8;
+    bstr_xappend_asprintf(talloc_ctx, text, "\033[0;%d%dm", 3 + (c >> 3) * 6, c & 7);
```

Comment:
> Also bright white is a worse color since it's usually pure white while regular white is light gray.

So what do you suggest? We disallow black and white? It really depends on the terminal color pallete.

Reply:
Or we do
```
// black -> gray
if (c == 0)
        c += 8;
// white -> gray
if (c == 15)
        c -= 8;
```

Reply:
> We only do it for black -> bright black.

Didn't see your response. We also want to do it for all other colors, because modules are randomly colored so 16 choices is better than 8

Reply:
I've changed it now, see if you like it.

Reply:
wdym?

Reply:
> but c == 15 results in 315 and 915

It is/was not possible in any of the version I pushed.